src/nsrt_learning/option_learning.py

```diff
@@ -267,26 +270,37 @@ class _LearnedNeuralParameterizedOption(ParameterizedOption):
     it was in the initial state. To handle this, we save the *absolute* desired
     state in the memory of the option during initialization, and compute the
     updated delta on each call to the policy.
+
+    The parameterized kwarg is for a baseline that learns a policy without
```

Comment:
rename to `is_parameterized` and change docstring to "The kwarg `is_parameterized` is for..." (it's currently confusing)